package.json

```diff
@@ -76,6 +76,7 @@
   },
   "dependencies": {
     "@capsizecss/core": "3.0.0",
+    "@ethereumjs/common": "4.1.0",
```

Comment:
I think you can skip this since it was already a sub dep  (see https://github.com/rainbow-me/browser-extension/pull/1346/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deL717)

This might remove a bunch of warnings from socket

Reply:
Yeah, seems like it's working fine without it.